xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -99,11 +136,15 @@
         <Parameter Name="assemblyPath" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="assemblyPath">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="assemblyPath">The path to the assembly.</param>
+        <summary>Gets an <see cref="T:System.Reflection.AssemblyName" /> for the assemblyPath.</summary>
```

Comment:
I assume the `assemblyPath` needs to be somehow "quoted" or something to format it correctly as an identifier. Probably `<paramref>`.

Reply:
I do not think it should be a param ref in the summary.  I'll leave this to @rpetrusha to determine

Reply:
I think I should change it to "assembly path"